src/main/java/com/vention/stock_market_share/controller/StockController.java

```diff
@@ -0,0 +1,31 @@
+package com.vention.stock_market_share.controller;
+
+import com.vention.stock_market_share.response.AlphaVintageResponse;
+import com.vention.stock_market_share.service.AlphaVintageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
```

Comment:
use @RequestMapping("/stock")
and delete ("/searchStockBySymbol") and ("/searchAllTradingShares")

For All just @GetMapping, for one use @GetMapping("/{symbol}") like PathVariable

Reply:
Done